cluster/environments/azure-stateful-keyvault/README.md

```diff
@@ -0,0 +1,3 @@
+# azure-stateful-keyvault
+
+The `azure-stateful-keyvault` environment restores a single production level AKS cluster from a Velero backup using the Velero Terraform module.
```

Comment:
I think more importantly, `azure-stateful-keyvault` defines a template for a cluster that includes stateful workloads that you want to backup and restore.